src/applications/Job.java

```diff
@@ -49,10 +49,37 @@ public void setArrivalTime(int arrivalTime) {
     public int getId() {
         return id;
     }
-
+    /**
+     * put the next job on the machine queue.
+     * @param machineShopSimulator
+     * @param p // the machine to get
```

Comment:
You shouldn't put a comment here; that way, the param is labeled for anything that looks at function docs (including mousing over the function name, for many modern IDEs.  It _might_ actually still work, just with a pair of slashes at the beginning, but that's still a little unnecessarily ugly.

Reply:
Can you explain this more?